app/views/foreman_ansible/job_templates/run_openscap_scans_-_ansible_default.erb

```diff
@@ -0,0 +1,20 @@
+<%#
+name: Run OpenSCAP scans - Ansible Default
+job_category: OpenSCAP Ansible Commands
+description_format: Run scan for all OpenSCAP policies on given hosts
+snippet: false
+provider_type: Ansible
+kind: job_template
+model: JobTemplate
+%>
+
+<% raise "Create and assign a policy to this host before proceeding" if @host.policies_enc_raw.empty? -%>
+---
+- hosts: all
+  tasks:
+<% @host.policies_enc_raw.map do |policy| -%>
```

Comment:
`map` collects results into an array, which is unnecessary here, `each` would be better.

Reply:
Done. Thanks once again.